inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -987,7 +988,8 @@
             operator: "MEMBER_OF",
             values: ["tag1", "tag2"]
           }
-        ]
+        ],
+        status: "STAGED"
```

Comment:
Should some of these be `STAGED` and some `APPLIED` for a more thorough test case?

Reply:
We don't have an integration test that applies rules yet. That's what I'm working on adding to cypress in [this PR](https://github.com/chef/automate/pull/870). So I can add some staged vs. applied assertions there